applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,191 @@
+    "max_iterations"                   : 30,
+    "number_cycles"                    : 1,
+    "reduction_factor"                 : 1.0,
+    "increase_factor"                  : 1.0,
+    "desired_iterations"               : 4,
+    "max_radius_factor"                : 10.0,
+    "min_radius_factor"                : 0.1,
+    "calculate_reactions"              : true,
+    "max_line_search_iterations"       : 5,
+    "first_alpha_value"                : 0.5,
+    "second_alpha_value"               : 1.0,
+    "min_alpha"                        : 0.1,
+    "max_alpha"                        : 2.0,
+    "line_search_tolerance"            : 0.5,
+    "rotation_dofs"                    : true,
+    "linear_solver_settings": {
+      "solver_type"                    : "LinearSolversApplication.sparse_lu",
+      "scaling"                        : true
+    },
+    "problem_domain_sub_model_part_list": ["Parts_Solid_Grond"],
+    "processes_sub_model_part_list"     : ["gravity", "WATER_PRESSURE_Initial", "WATER_PRESSURE_Bottom", "WATER_PRESSURE_Top"],
+    "body_domain_sub_model_part_list"   : ["Parts_Solid_Grond"],
+    "newmark_beta"                     : 0.25,
+    "newmark_gamma"                    : 0.5,
+    "newmark_theta"                    : 0.5,
+    "rayleigh_m"                       : 0.0,
+    "rayleigh_k"                       : 0.0,
+    "move_mesh_flag"                   : false
+  },
+  "output_processes": {
+    "gid_output": [
+      {
+        "python_module"                : "gid_output_process",
+        "kratos_module"                : "KratosMultiphysics",
+        "process_name"                 : "GiDOutputProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.porous_computational_model_part",
+          "output_name"                : "stage1",
+          "postprocess_parameters": {
+            "result_file_configuration": {
+              "gidpost_flags": {
+                "WriteDeformedMeshFlag": "WriteUndeformed",
+                "WriteConditionsFlag"  : "WriteElementsOnly",
+                "GiDPostMode"          : "GiD_PostAscii",
+                "MultiFileFlag"        : "SingleFile"
+              },
+              "file_label"             : "step",
+              "output_control_type"    : "step",
+              "output_interval"        : 1,
+              "body_output"            : true,
+              "node_output"            : false,
+              "skin_output"            : false,
+              "plane_output"           : [],
+              "nodal_results"          : ["DISPLACEMENT", "TOTAL_DISPLACEMENT", "WATER_PRESSURE", "VOLUME_ACCELERATION",
+			                              "LINE_LOAD_Y", "TOTAL_STRESS_TENSOR", "CAUCHY_STRESS_TENSOR"],
+              "gauss_point_results"    : ["GREEN_LAGRANGE_STRAIN_TENSOR", "ENGINEERING_STRAIN_TENSOR", "VON_MISES_STRESS",
+                                          "FLUID_FLUX_VECTOR", "HYDRAULIC_HEAD", "DEGREE_OF_SATURATION"]
```

Comment:
Please limit the requested output to what you actually want to check. This is particularly relevant for code coverage, as it may suggest that some output items are covered by tests, which in fact aren't.

Reply:
Removed most items. I would like to do without output process, but the test helper looks at it to determine the node numbers.